lib/Form/Type/ContentType/ContentTypeUpdateType.php

```diff
@@ -120,39 +92,27 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'entry_options' => ['languageCode' => $options['languageCode']],
                 'label' => 'content_type.field_definitions_data',
             ])
-            ->add('fieldTypeSelection', ChoiceType::class, [
-                'choices' => array_flip($this->getFieldTypeList()),
-                'choices_as_values' => true,
+            ->add('fieldTypeSelection', FieldTypeChoiceType::class, [
                 'mapped' => false,
                 'label' => 'content_type.field_type_selection',
             ])
-            ->add('addFieldDefinition', SubmitType::class, ['label' => 'content_type.add_field_definition'])
```

Comment:
Please don't reformat unless you need to. It makes the diff bigger for no real reason. It also applies to other lines in this PR.